src/data/blog/hiring-work-for-users.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Want to help our projects work for users?'
+slug: 'hiring-work-for-users'
+image: null
+featured: 0
+page: 0
+status: 'published'
+language: 'en_US'
+meta_title: null
+meta_description: null
+author_id: 1
+created_at: '2021-07-09T13:29:43.000Z'
```

Comment:
I think this needs a published_at as well, otherwise it defaults to 01-01-1970

Reply:
Neat - added that in. Thx.